scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -67,9 +71,21 @@ export function RelatedItemsDirective(
             scope.onCreated = (items: Array<IArticle>) => {
                 items.forEach((item) => {
                     scope.addRelatedItem(item);
+                    storage.setItem(item._id, scope.item._id);
```

Comment:
It's not good to use item ID as a storage key because it may be used for different purposes from other places in the app. It's not readable too if I saw it in the browser, I wouldn't be able to tell where is it getting set/used in the code.

It's better to use something like `open-item-after-related-closed--${id}`